requirements/test.txt

```diff
@@ -1,2 +1,3 @@
 pytest
 pytest-cov
+idom
```

Comment:
So the nice thing is that you don't actually need to add this as a dependency since we never actually execute `tests/hook_usage_test_cases.py`

Reply:
That also artificially made me have to add `getattr(node, "id", None) == "component"`, just for tests to pass. Maybe the syntax tree doesn't expand out due to no imports, so `getattr(getattr(node, "value", None), "id", None) == "idom" and getattr(node, "attr", None) == "component"` isn't always true.